.github/workflows/ci.yml

```diff
@@ -192,4 +192,38 @@ jobs:
           source cacitesting.env
           python -m unittest discover -s tests/skr
 
+  test-encfs:
+    name: Test EncFS
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install c_aci_testing package
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: ./scripts/install-c-aci-testing.sh
```

Comment:
If you merge setup and this script, can change this to setup.sh

Reply:
setup.sh is more for local development, for ci, we only need the testing package so I think it makes sense to keep them separate